contracts/ForkableBridge.sol

```diff
@@ -57,7 +57,7 @@ contract ForkableBridge is
         address to
     ) external onlyAfterForking {
         require(_hardAssetManager == msg.sender, "Not authorized");
-        require(to == children[0] || to == children[1], "Invalid to address");
+        require(to == children[0] || to == children[1], "Invalid to");
```

Comment:
These changes were needed to cut contract size

Reply:
> Now, the users just have the option to either mint both child-tokens after the forking process - or only one child-fork token, if the second version might have a broken minting functionality.

OK, that makes sense. I was going to suggest that.

Reply:
> These changes were needed to cut contract size

Maybe not right now but would it help if we replaced the error strings with solidity errors with the strings in Natspec? Or would that make things worse?

Reply:
yes, let's do that later. It would help a little

Reply:
OK, made an issue for that. Probably best to do it project-wide at some point so let's leave the strings for now.

https://github.com/RealityETH/subjectivocracy/issues/102